.github/workflows/issue-sync.yml

```
@@ -0,0 +1,268 @@
+    name: Sync TODO.md → GitHub Issues
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    permissions:
+      contents: write
+      issues: write
+
+    steps:
+      - name: Check commit author
+        id: check-author
+        run: |
```

actionlint (1.7.10)

[error] "github.event.head_commit.author.name" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)